tests/unit/objects/test_groups.py

```diff
@@ -466,3 +491,9 @@ def test_delete_saml_group_link(group, resp_delete_saml_group_link):
 
 def test_group_restore(group, resp_restore_group):
     group.restore()
+
+
+def test_create_group_service_account(group, resp_create_group_service_account):
+    group.service_accounts.create(
+        {"name": "gitlab-service-account", "username": "gitlab-service-account"}
+    )
```

Comment:
I know it's not testing much, but if we already mock the response content should we also assert on this? Same as in the `test_create_group` test above.

```suggestion
    account = group.service_accounts.create(
        {"name": "gitlab-service-account", "username": "gitlab-service-account"}
    )
    assert account.name == "gitlab-service-account"
    assert account.username == "gitlab-service-account"
```